manifests/helper/systemd.pp

```diff
@@ -0,0 +1,30 @@
+# Definition: systemd
+#
+#  Lets create some Systemd scripts
+#
+define st2::helper::systemd (
+  $st2_process  = undef,
+  $process_type = 'single'
+  ) {
+
+  if $process_type == 'multi' {
+    $type = "${process_type}@"
+  } else {
+    $type = ''
+  }
+
+  file{"/etc/systemd/system/${st2_process}${type}.service":
```

Comment:
I tweaked it to only enable the single process ones.  This should avoid any issues with the @ ones vs the singles.